frank-doc-doclet/src/test/java/org/frankframework/frankdoc/testtarget/wrapper/annotations/of/annotations/Annotation.java

```diff
@@ -0,0 +1,22 @@
+package org.frankframework.frankdoc.testtarget.wrapper.annotations.of.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Copied from IAF project.
+ * 
+ * @author Jaco de Groot
+ */
```

Comment:
Kan je hier beschrijven wat deze annotatie doet, of waar hij voor gebruikt wordt?

Reply:
Gedaan.